ui/components/multichain/network-list-menu/network-list-menu.tsx

```diff
@@ -277,6 +279,8 @@ export const NetworkListMenu = ({ onClose }: { onClose: () => void }) => {
             network.rpcEndpoints[network.defaultRpcEndpointIndex];
           dispatch(setActiveNetwork(networkClientId));
           dispatch(toggleNetworkMenu());
+          dispatch(updateCustomNonce(''));
```

Comment:
Minor, is this handler big enough to warrant a `handleClick` lambda or function?

Reply:
I would prefer refactoring this in separate PR.